tests/json-fixtures/test_transactions.py

```diff
@@ -80,12 +78,15 @@ def test_transaction_fixtures(fixture_name, fixture):
 
         sender = to_canonical_address(fixture['sender'])
 
-        assert transaction.sender == sender
+        try:
+            assert transaction.sender == sender
+        except BadSignature:
+            assert not (27 <= transaction.v <= 34)
     else:
         # check RLP correctness
         try:
             transaction = rlp.decode(fixture['rlp'], sedes=TransactionClass)
-        except rlp.exceptions.ObjectDeserializationError:
+        except (rlp.exceptions.ObjectDeserializationError, rlp.exceptions.DecodingError, TypeError):
```

Comment:
@pipermerriam I've attached the tests that are passing due to these exceptions, most of them have WrongRLP in the title and I believe the ones that don't should be caught here.  I've attached a list of the errors that are passing due to the aforementioned exceptions.
[passing_due_to_rlp_exception.txt](https://github.com/pipermerriam/py-evm/files/1170218/passing_due_to_rlp_exception.txt)